### PR TITLE
Update index.adoc

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
We had line breaks instead of paragraphs, so I made my best guess as to how to fix.
Part of unplanned work: https://www.mulesoft.org/jira/browse/DOCS-11498

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -3,15 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Security provides a layered approach to secure your application network. These layers work together to protect both the application network and the network's individual nodes by controlling access to APIs, enforcing policies, and proxying all inbound or outbound traffic to mitigate external threats and attacks. +
-Anypoint Security provides you with a dedicated endpoint to detect attacks and validate traffic without taxing your network implementations.
+Anypoint Security provides a layered approach to secure your application network. These layers work together to protect both the application network and the network's individual nodes by controlling access to APIs, enforcing policies, and proxying all inbound or outbound traffic to mitigate external threats and attacks.
```
Comment (valkyrie69) [inline modules/ROOT/pages/index.adoc:6]: There is no line break or new <p> at all now--this is all one para. If you want a line break, you need one more line. If you meant this to be just one para, disregard and carry on. :-)

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -3,15 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Security provides a layered approach to secure your application network. These layers work together to protect both the application network and the network's individual nodes by controlling access to APIs, enforcing policies, and proxying all inbound or outbound traffic to mitigate external threats and attacks. +
-Anypoint Security provides you with a dedicated endpoint to detect attacks and validate traffic without taxing your network implementations.
+Anypoint Security provides a layered approach to secure your application network. These layers work together to protect both the application network and the network's individual nodes by controlling access to APIs, enforcing policies, and proxying all inbound or outbound traffic to mitigate external threats and attacks. 
+Anypoint Security also provides you with a dedicated endpoint to detect attacks and validate traffic without taxing your network implementations.
 
 image::index-1c19d.png[]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index.adoc]: image needs alt text for accessibility. That may be beyond the scope of this PR and just something to note.

**Review by valkyrie69 — DISMISSED** [review]

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/index.adoc:6]: Yah, I think the plus sign was left in by accident.

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/index.adoc]: great catch. I hate doc debt, so fixing.

Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/index.adoc]:
```suggestion
image::index-1c19d.png[Anypoint Security diagram: Mule apps in Runtime Fabric protected by the request endpoint with policies and TLS validation.]
```

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/index.adoc]: 122 chars

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**